Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/MessageDialogSamplePage.xaml

```diff
@@ -1,103 +0,0 @@
-﻿<Page x:Class="Uno.Gallery.Views.Samples.MessageDialogSamplePage"
```

Comment:
@MartinZikmund is there a reference to docs where it recommends not using MessageDialog? I don't agree that this is a good idea, given it's by far the simplest way to display messages to the user - ContentDialog has a lot of overhead just to communicate a simple message

Reply:
We discussed earlier, but for future reference here in the docs: https://learn.microsoft.com/en-us/uwp/api/windows.ui.popups.messagedialog?view=winrt-22621#definition